kotest-framework/kotest-framework-engine/src/commonMain/kotlin/io/kotest/engine/test/interceptors/TestEnabledCheckInterceptor.kt

```diff
@@ -32,6 +35,7 @@ internal class TestEnabledCheckInterceptor(private val configuration: ProjectCon
          }
          false -> {
             logger.log { Pair(testCase.name.testName, "Test is disabled: ${enabled.reason}") }
+            testExtensions.disabledTestListenersInvocation(testCase, enabled.reason)
```

Comment:
We prob want to fire this in the same place we fire the test engine listener, so we capture all places ?

Reply:
I this yes, we want this.